Block/Adminhtml/Items/Column/Qty.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rvvup\Payments\Block\Adminhtml\Items\Column;
+
+use Magento\Backend\Block\Template\Context;
+use Magento\Catalog\Model\Product\OptionFactory;
+use Magento\CatalogInventory\Api\StockConfigurationInterface;
+use Magento\CatalogInventory\Api\StockRegistryInterface;
+use Magento\Framework\Registry;
+use Magento\Sales\Model\Order\Item;
+use Rvvup\Payments\Model\PendingQty;
+
+class Qty extends \Magento\Sales\Block\Adminhtml\Items\Column\Qty
+{
+    /**
+     * @var mixed
+     */
+    private $pendingQtyService;
+
+    /**
+     * @param Context $context
+     * @param StockRegistryInterface $stockRegistry
+     * @param StockConfigurationInterface $stockConfiguration
+     * @param Registry $registry
+     * @param OptionFactory $optionFactory
+     * @param PendingQty $pendingQtyService
+     */
+    public function __construct(
+        Context $context,
+        StockRegistryInterface $stockRegistry,
+        StockConfigurationInterface $stockConfiguration,
+        Registry $registry,
+        OptionFactory $optionFactory,
+        PendingQty $pendingQtyService
+    ) {
+        $this->pendingQtyService = $pendingQtyService;
+        parent::__construct($context, $stockRegistry, $stockConfiguration, $registry, $optionFactory);
+    }
+
+    public function getQtyRvvupPendingRefund(Item $item): float
```

Comment:
just noticed that the return type from here is float. does this need to deal with payment methods like crypto where the amounts can be quite small fractions?

Reply:
It is actually quantity, it is always int , just comes as string, will cast it to tint